app_template_dev.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""Template for CDK Application"""
```

Comment:
What is this meant to be used for?

Reply:
So the idea here is that we don't run the risk of committing account numbers, hard-coded initials, etc. So when doing development work we make a copy of this template, rename it, and deploy from there instead of in app.py.

Reply:
That makes sense! I added a suggestion to make it more clear in the docs. Might I also suggest that it retrieve the sds_id from cdk.json and have the user change their local copy of that instead of having to remember to update here? Basically just remove the .join and add a note in the docs about changing the deploy name.